src/DotVVM.Samples.Common/ViewModels/FeatureSamples/PostbackConcurrency/RedirectPostbackQueueViewModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotVVM.Samples.Common.ViewModels.FeatureSamples.PostbackConcurrency
+{
+    public class RedirectPostbackQueueViewModel : RedirectPostbackQueueMasterViewModel
+    {
+
+        public int Value { get; set; }
+
+        public override async Task Init()
+        {
+            if (Context.Query.ContainsKey("time"))
```

Comment:
Could you please wait only for the first load. Otherwise, manual clicking on the sample may be quite annoying

Reply:
Fixed - I didn't expect anyone to click the buttons after the redirect - the UI tests use the buttons only before the redirect. But for humans, it'll be easier.